### PR TITLE
Experimental tools

### PR DESCRIPTION
**Pull request by edpaget**

### CODE REVIEW

File: app/pages/lab/workflow.cjsx [hunk 1]
```diff
@@ -33,6 +33,9 @@ EditWorkflowPage = React.createClass
     viewQuery = workflow: @props.workflow.id, reload: @state.forceReloader
     @makeHref 'project-classify', viewParams, viewQuery
 
+  canUseTask: (owner, project, task)->
+    task in project.experimental_tools or owner.admin or owner.display_name is 'brian-testing'
```
Comment (brian-c) [inline app/pages/lab/workflow.cjsx]: Can we just make this `task in project.experimental_tools`? That'd leave one right way to enable experimental tools.

Comment (edpaget) [inline app/pages/lab/workflow.cjsx]: Sure

Comment (edpaget) [inline app/pages/lab/workflow.cjsx]: Done

**Comment by brian-c** [comment]
Looks good!

**Comment by aliburchard** [comment]
Is this deployed yet? I can't seem to find it (and am not sure if I'm just looking in the wrong place).

**Comment by edpaget** [comment]
@aliburchard in the admin panel on the project status page.

**Comment by aliburchard** [comment]
Thanks! didn't realize i had to search for the project from the admin/status page, sorry.

**Pull request merged**